support-workers/src/main/scala/com/gu/support/workers/lambdas/SendAcquisitionEvent.scala

```diff
@@ -38,7 +41,13 @@ class SendAcquisitionEvent(serviceProvider: ServiceProvider = ServiceProvider)
     services.acquisitionService.submit(
       SendAcquisitionEventStateAndRequestInfo(state, requestInfo)
     ).fold(
-      errors => throw AnalyticsServiceErrorList(errors), _ => HandlerResult(Unit, requestInfo)
+      errors => throw AnalyticsServiceErrorList(errors),
+      _ => {
+        val cloudwatchEvent = paymentSuccessRequest(Configuration.stage, paymentProviderFromPaymentMethod(state.paymentMethod), state.product)
+        AwsCloudWatchMetricPut(client)(cloudwatchEvent)
+
+        HandlerResult(Unit, requestInfo)
```

Comment:
`Unit` is a type! I realise it's not your change, but the value of `Unit` is `()`

Reply:
good spot